.travis.yml

```diff
@@ -15,11 +15,13 @@ before_install:
   - docker rm -f mysql || true          # in case the after_success step does not run
   - docker pull mysql/mysql-server:5.6
   # port 3306 is not available because Travis runs its own MySQL after 2006-12-01
-  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.6
-
+  - docker run -p 3307:3306 --name mysql -e MYSQL_ROOT_PASSWORD=rawls-test -e MYSQL_USER=rawls-test -e MYSQL_PASSWORD=rawls-test -e MYSQL_DATABASE=testdb -d mysql/mysql-server:5.7
```

Comment:
Be careful to make sure you have the right MySQL version here. It's been downgraded recently to match the version in production so I suspect 5.6 might be the correct one now (please double-check).